ui/src/app/pages/signed-in/access-renewal-notification.tsx

```diff
@@ -37,6 +41,16 @@ export const AccessRenewalNotificationMaybe = (
       ? daysRemaining + ' days remaining.'
       : 'Your access has expired.';
 
+  const boxColor = moreThan14daysRemaining
```

Comment:
nested trinaries are confusing - could you use something else here, like a cond?